File: Mathematical/pascals_triangle.py
def printPascal(n):
	for row in range(1, n + 1):
		ct = 1
		for i in range(1, row + 1):
			print(ct, end=" ")
			ct = int(ct * (row - i) / i)
		print("")
printPascal(int(input("Enter number of levels: ")))

File: Mathematical/pascals_triangle.js

function printPascal(n){
    for(let row = 1; row < n + 1; row++)
    {
        let ct = 1;
        let store = "1";
        for(let i = 1; i < row;i++)
        {
            ct = Math.trunc(ct * (row - i) / i);
            store += " " + ct;
        }
        console.log(store);
    };
}
     
function printRowRPascal(r){
        let ct = 1;
        let store = "1";
        for(let i = 1; i < r;i++)
        {
            ct = Math.trunc(ct * (r - i) / i);
            store += " " + ct;
        }
        console.log(store);
}
       
